.github/workflows/test_pr_changes.yml

```diff
@@ -2,7 +2,7 @@ name: Test Pull Request Changes
 
 on: 
   pull_request:
-    types: [opened, synchronize, edited]
+    types: [opened, synchronize, edited, ready_for_review]
```

Comment:
When a PR is created as draft and then turned into "ready for review", the tests have not been executed and thus a PR could potentially be merged with failing tests.